cpp/unittests/gsdkTests.cpp

```diff
@@ -168,6 +168,52 @@ namespace Microsoft
                     Assert::IsTrue(GSDKInternal::m_instance->m_heartbeatRequest.m_currentGameState == GameState::Active, L"Verify state was changed.");
                 }
 
+                TEST_METHOD(DecodeAgentResponseJsonDoesntCrash)
```

Comment:
I would put an _ in between (i think that's the convention we are following. So it would be DecodeAgentResponse_JsonDoesntCrash.

Similar for the one below (you don't have to change existing ones unless you want to :))